schedule_manager.py

```diff
@@ -0,0 +1,467 @@
+#!/usr/bin/env python3
+"""
+Schedule Manager for Goose Perception
+Manages scheduled jobs using GooseSchedule instead of hard-coded cron logic
+"""
+
+import subprocess
+import json
+import sys
+import os
+from typing import Dict, List, Optional, Tuple
+from dataclasses import dataclass
+from pathlib import Path
+
+@dataclass
+class ScheduleConfig:
+    """Configuration for a scheduled recipe"""
+    recipe_path: str
+    frequency: str
+    weekday_only: bool = False
+    time_of_day: Optional[str] = None  # morning, afternoon, evening
+    description: str = ""
+
+class ScheduleManager:
+    """Manages GooseSchedule jobs for goose-perception recipes"""
+    
+    def __init__(self):
+        self.observers_dir = Path("observers")
+        self.local_observers_dir = Path("observers/local-observers")
+        
+        # Define all scheduled recipes with their configurations
+        self.schedules = {
+            # Main observation recipes
+            "work-summary": ScheduleConfig(
```

Comment:
could we enhance this so instead of repeating things, we load everything in observers etc - and then read a field which has the schedule from it?

Reply:
(that way we don't need to forget to add things here) - or have a structure like we do when we adapt recipes?